admin_guide/quota.adoc

```diff
@@ -476,3 +476,48 @@ If a quota has been defined for your project, see
 xref:../dev_guide/deployments/basic_deployment_operations.adoc#deployment-resources[Deployment Resources]
 for considerations on any deployment configurations.
 endif::[]
+
+ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
+[[limited-resources-quota]]
+== Configuring resources whose consumption is limited by default
```

Comment:
The title seems a bit confusing to me. It sounds as if resource limiting is the default, rather than resource limiting by default needs to be configured.
"Configuring a resource to be limited by default" (?)